src/drivers/dw/dma.c

```diff
@@ -273,6 +273,13 @@ static int dw_dma_start(struct dma_chan_data *channel)
 
 	irq_local_disable(flags);
 
+	/*
```

Comment:
@ktrzcinx what makes you think the SSP is managed with a timer-based scheduling? It's clearly not true for legacy platforms. I also don't know what this means for SSP slave cases.

Reply:
>SSP is managed with a timer-based scheduling

I didn't notice anything about timers, just about time. I would like to highlight time restrictions related with DAI interfaces (putting new data portion is regulated by clock, so it's scheduled in regular time periods) in opposite to dma which one works on demand without restrictions measured in time units.